src/regex-transform-stream.ts

```diff
@@ -1,3 +1,4 @@
+import { TransformStream } from "web-streams-polyfill"
```

Comment:
I seem to recall this breaks stuff for WebSerial. I had a local change that just did:

```ts
import "web-streams-polyfill/es2018"
```

here instead. I don't currently have access to my AxiDraw hardware, could you test that this works correctly both through WebSerial and through node-serial?

Reply:
Sure, I'll check when I get the chance.  I'm trying to untangle the broken dependencies first.

Reply:
> Sure, I'll check when I get the chance. I'm trying to untangle the broken dependencies first.

Works with node-serial.  I haven't gotten WebSerial to work yet.